client/my-sites/plugins/plugin-details-CTA/manage-site-plugins-dialog.jsx

```diff
@@ -0,0 +1,81 @@
+import { FEATURE_INSTALL_PLUGINS } from '@automattic/calypso-products';
+import { Dialog, Button } from '@automattic/components';
+import { useTranslate } from 'i18n-calypso';
+import { useSelector } from 'react-redux';
+import PluginSiteList from 'calypso/my-sites/plugins/plugin-site-list';
+import { siteObjectsToSiteIds } from 'calypso/my-sites/plugins/utils';
+import { getBillingInterval } from 'calypso/state/marketplace/billing-interval/selectors';
+import {
+	getSiteObjectsWithPlugin,
+	getSiteObjectsWithoutPlugin,
+} from 'calypso/state/plugins/installed/selectors';
+import getSelectedOrAllSitesWithPlugins from 'calypso/state/selectors/get-selected-or-all-sites-with-plugins';
+import siteHasFeature from 'calypso/state/selectors/site-has-feature';
+
+export function ManageSitePluginsDialog( { isVisible, onClose, plugin } ) {
```

Comment:
Can we use an arrow function here to increase parity with the rest of the codebase?

Reply:
Done at https://github.com/Automattic/wp-calypso/pull/66282/commits/baea81a75bd82b58401be5a46abba55555105248